content/en/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md

```diff
@@ -68,7 +68,8 @@ on Kubernetes dual-stack support see [Dual-stack support with kubeadm](/docs/set
    ExecStart=
    # Replace "systemd" with the cgroup driver of your container runtime. The default value in the kubelet is "cgroupfs".
    # Replace the value of "--container-runtime-endpoint" for a different container runtime if needed.
-   ExecStart=/usr/bin/kubelet --address=127.0.0.1 --pod-manifest-path=/etc/kubernetes/manifests --cgroup-driver=systemd --container-runtime=remote --container-runtime-endpoint=unix:///var/run/containerd/containerd.sock
+   ExecStart=/usr/bin/kubelet --address=127.0.0.1 --pod-manifest-path=/etc/kubernetes/manifests --cgroup-driver=systemd 
+   --container-runtime-endpoint=unix:///var/run/containerd/containerd.sock
```

Comment:
LGTM

Reply:
This looks like 2 lines; the second line is ` --container-runtime-endpoint=unix:///var/run/containerd/containerd.sock`. Does that really work in a systemd config?

Reply:
It should be one line. Updated.